packages/ui-kit/src/components/PieChart/PieChart.module.scss

```diff
@@ -0,0 +1,87 @@
+@use '../../themes/typography';
```

Comment:
It doesn't seem that the file is properly formatted. Please install `SCSS Formatter` extension and try to re-save your scss files. It should apply the formatting.
Also, please add this extension (its id is `sibiraj-s.vscode-scss-formatter`) to the recommended extensions [.vscode/extensions.json](https://github.com/propeldata/ui-kit/blob/main/.vscode/extensions.json)

<img width="1359" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/4e17fed4-b030-40ec-b1c8-fd10b937dc09">